tests_v2/test_ssh_remote_port_detection.py

```
@@ -0,0 +1,342 @@
+            _must(
+                REMOTE_HTTP_PORT in listening_ports,
+                f"workspace.list should include listening port {REMOTE_HTTP_PORT}: {row}",
+            )
+
+            if surface_id:
+                client.send_key_surface(surface_id, "ctrl-c")
+            if workspace_id:
+                try:
+                    client.close_workspace(workspace_id)
+                except Exception:
+                    pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)